app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -277,26 +278,38 @@ class DefaultToolbarMenu(
         onItemTapped.invoke(ToolbarMenu.Item.Quit)
     }
 
-    private val readerMode = BrowserMenuImageSwitch(
+    private val readerMode = BrowserMenuHighlightableSwitch(
         label = context.getString(R.string.browser_menu_read),
-        imageResource = R.drawable.ic_readermode,
-        initialState = readerModeStateProvider
+        startImageResource = R.drawable.ic_readermode,
+        initialState = readerModeStateProvider,
+        highlight = BrowserMenuHighlight.LowPriority(
+            label = context.getString(R.string.browser_menu_read),
+            notificationTint = getColor(context, R.color.whats_new_notification_color)
+        )
     ) { checked ->
         onItemTapped.invoke(ToolbarMenu.Item.ReaderMode(checked))
     }
 
-    private val readerAppearance = BrowserMenuImageText(
+    private val readerAppearance = BrowserMenuHighlightableItem(
```

Comment:
Does appearance also need a highlight? It feels redundant.

Reply:
@apbitner what do you think?  I have no strong feelings either way on this one.

Reply:
I don't think it's necessary. 'Appearance' may not live in the toolbar menu permanently anyways.

Reply:
Thanks Adam, I'll pull it out.